README.md

```diff
@@ -236,7 +236,7 @@ React will automatically check the propTypes you set on the component, but if
 you are using PropTypes without React then you may want to manually call
 `PropTypes.checkPropTypes`, like so:
 
-```
+```js
```

Comment:
Why 'jsx' above and just 'js' here? This is still better than no syntax highlighting of course.